src/index.ts

```diff
@@ -132,77 +130,90 @@ async function createRegularSprite(): Promise<void> {
   let svgs;
   // Add all SVGs to sprite
   try {
-    svgs = fs.readdirSync(fullPath);
+    svgs = fs.readdirSync(path);
   } catch (err) {
     throw err;
   }
   for (const item of svgs) {
     let file;
     let optimizedFile;
 
-    if(item.endsWith('.svg')) {
+    if (item.endsWith('.svg')) {
       try {
-        const path = join(fullPath, item);
+        const svgPath = join(path, item);
 
-        file = fs.readFileSync(path, { encoding: 'utf-8' });
+        file = fs.readFileSync(svgPath, { encoding: 'utf-8' });
         if (!file) {
           console.log(chalk.yellow(`Skipping ${item}, because the file is empty...`));
           continue;
         }
-        const styleConvertedFile = await svgoConvertStyles.optimize(file, { path });
+
+        const styleConvertedFile = await svgoConvertStyles.optimize(file, { path: svgPath });
         optimizedFile = await svgoRemoveStyles.optimize(styleConvertedFile.data);
-        spriter.add(resolve(path), '', optimizedFile.data);
+        spriter.add(resolve(svgPath), '', optimizedFile.data);
         svgCount++;
       } catch (err) {
         throw err;
       }
     }
   }
 
-  if(svgCount) {
+  if (svgCount) {
     console.log(chalk.green(svgs.length) + chalk.grey(' SVGs found.'));
   } else {
-    throw new Error(`No SVG files found in '${fullPath}'. Make sure you are using the correct path.`)
+    throw new Error(`No SVG files found in '${path}'. Make sure you are using the correct path.`)
   }
   // Compile the sprite
-  spriter.compile(function(err: Error, result: Array<PlainObjectType>): void {
+  spriter.compile((err: Error, result: Array<PlainObjectType>): void => {
     if (err) {
       throw err;
     }
     // @Todo(Chris): check if this is ever used
     // this has no effect as far as i can see
+    console.log(result)
```

Comment:
remove console.log

Reply:
Ah, I see you left a todo for me, sorry. What do you mean, has no effect? The result contains information for svg-sprite on which modes were set active in the options (currently we only use symbol mode). https://www.npmjs.com/package/svg-sprite#usage-pattern

Reply:
Ah I see, this comment isn't from you... just delete it with the console.log all together :D I think this was a leftover from the typings merge